client/js/components/faq/DpFaqSupport.vue

```diff
@@ -0,0 +1,118 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div
+    id="scFaq"
+    class="c-support-wrapper space-inset-m">
+    <h2 class="font-normal">
+      {{ Translator.trans('support.heading') }}
+    </h2>
+    <p class="mb-5">
+      {{ Translator.trans('support.introduction') }}
+    </p>
+    <p>
+      {{ Translator.trans('support.contact.advice') }}
+    </p>
+    <h3 class="mt-5">
+      {{ Translator.trans('support.content') }}
+    </h3>
+    <ul class="grid lg:grid-cols-3 gap-3">
+      <li
+        v-for="contact in contacts"
+        :v-key="contact.id"
+        class="space-inset-m h-48 c-support-card">
+        <dp-faq-support-card>
+          <template v-slot:title>
+            <h4 class="font-semibold">
+              {{ contact.attributes.title }}
+            </h4>
+          </template>
+          <template v-slot:phonenumber>
+            <p class="mt-3 mb-0 inline-block font-semibold">
+              {{ contact.attributes.phoneNumber }}
+            </p>
+          </template>
+          <template v-slot:email>
+            <p class="mb-0 font-normal">
+              {{ contact.attributes.eMailAddress }}
+            </p>
+          </template>
+          <template v-slot:reachability>
+            <p class="mt-4 lg:mt-2 font-normal">
+              {{ (contact.attributes.text).replace(/<(?:"[^"]*"['"]*|'[^']*'['"]*|[^'">])+>/g, "") }}
```

Comment:
why do You need this here?

Reply:
this is to replace double p tag in the mark up (changed this code here for replacing only p tags). the one p tag is reproduced automatically by the editor and I tried to avoid this, otherwise you would see this in the textfield